designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.html

```diff
@@ -73,7 +73,6 @@ <h3 ng-if="$ctrl.project.value.projectType != 'other'" style="text-align: center
         <div class="prj-head-container">
             <span class="prj-head-title">
                 {{$ctrl.project.value.projectId}} | <strong>{{ $ctrl.project.value.title }}</strong>
-                <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)"><strong>Edit Project</strong></a>
             </span>
```

Comment:
Unless Craig and Maria said to remove this here, we should Keep the Edit Project button and modal in place here. I would just add the formatting changes you had earlier so that it's right aligned.